### PR TITLE
flatten the topology search when filtering by team

### PR DESCRIPTION
**Pull request by adityathebe**

### CODE REVIEW

File: src/pages/TopologyPage.tsx [hunk 1]
```diff
@@ -104,13 +104,17 @@ export function TopologyPage() {
...
+        flatten: false,
         hidden: showHiddenComponents === "no" ? false : undefined
       };
+
+      if (
+        topologyType.toLowerCase() !== "all" ||
+        team.toLowerCase() !== "all"
+      ) {
+        apiParams.flatten = true;
+      }
```
Comment (mainawycliffe) [inline src/pages/TopologyPage.tsx:116]:
```suggestion
        hidden: showHiddenComponents === "no" ? false : undefined,
        ...((topologyType.toLowerCase() !== "all" ||
          team.toLowerCase() !== "all") && {
          flatten: true
        })
```

Comment (mainawycliffe) [reply src/pages/TopologyPage.tsx:116]: I would probably do it like this but otherwise looks good.

**Review by mainawycliffe — APPROVED** [review]
Otherwise looks good.

**Comment by adityathebe** [comment]
We won't be flattening filters by team

**Pull request closed without merging**